labs/nft-auction.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: NFT Auction App
+parent: Labs
+---
+# Lab Name
+nft-auction
+
+# Short Description
+NFT Auction application lets user to create NFTs and auction them off to other users in a marketplace model using Hyperledger Fabric
+
+# Scope of Lab
+This lab uses hyperledger fabric as the underlying ledger technology to create a auction market. This helps the developer to vizualize how the NFT concept can be extended to private blockchains like Hyperledger Fabric
+The application uses Hyperledger 2.X version in its current form
```

Comment:
```suggestion
The application uses Hyperledger Fabric 2.X version in its current form
```

Reply:
Thanks for your input will incorporate in next commit